scripts/validate_experiment_record.py

```diff
@@ -1,250 +1,55 @@
-
-    # Rest of the samples
-    part = list()
-    for biosample_id in data:
-        part.append(target_dict[biosample_id])
-    total_results = get_validation_results(total_results, part, rulesets)
-    return total_results
-
-
-def get_validation_results(total_results, part, rulesets):
-    for ruleset in rulesets:
-        validation_results = validate_record(part, ruleset)
-        total_results = merge_results(total_results, validation_results, ruleset)
-    return total_results
-
-
-def validate_record(data, ruleset):
-    tmp_out_file = 'tmp_experiment_records.json'
-    with open(tmp_out_file, 'w') as w:
-        w.write("[\n")
-        for index, item in enumerate(data):
-            converted_data = convert_data(item)
-            converted_data = json.dumps(converted_data)
-            if index != 0:
-                w.write(",\n")
-            w.write(f"{converted_data}\n")
-        w.write("]\n")
-    try:
-        command = f'curl -s -F "format=json" -F "rule_set_name={ruleset}" -F "file_format=JSON"' + \
-                  f' -F "metadata_file=@{tmp_out_file}" "https://www.ebi.ac.uk/vg/faang/validate" > validation.json'
-        os.system(command)
-    except:
-        # TODO log to error
-        print("Validation Error!!!")
-        sys.exit(0)
-    with open('validation.json', 'r') as f:
-        data = json.load(f)
-    return parse_validation_results(data['entities'])
-
-
-def merge_results(total_results, validation_results, ruleset):
-    subresults = dict()
-    if ruleset in total_results:
-        subresults = total_results[ruleset]
-    if 'summary' in subresults:
-        for status in ['pass', 'warning', 'error']:
-            if status in subresults['summary'] and status in validation_results['summary']:
-                # TODO check this code
-                # subresults['summary'].setdefault(status, [])
-                subresults['summary'][status] += validation_results['summary'][status]
-            elif status in validation_results['summary']:
-                subresults['summary'][status] = validation_results['summary'][status]
-            else:
-                subresults['summary'][status] = 0
-        for tmp in list(validation_results['detail'].keys()):
-            subresults['detail'][tmp] = validation_results['detail'][tmp]
-        new_error_messages = validation_results['errors']
-        for msg in list(new_error_messages.keys()):
-            if msg in subresults['errors']:
-                subresults['errors'][msg] += new_error_messages[msg]
+import validate_record
+import utils
+from typing import List, Dict
+
+EXPERIMENT_FIELDS_CONVERSION_MAPPING = {
+}
+
+FIELDS_TO_BE_REMOVED = [
+    'accession',  # validation tool use fixed id field
+    # not in the ruleset
+    'standardMet',
+    'versionLastStandardMet'
+]
+
+logger = utils.create_logging_instance('validate_experiment')
+
+
+class validate_experiment_record(validate_record.validate_record):
```

Comment:
class name should use CamelCase convention

Reply:
fixed in 2db90ac